site/en/performance/quantization_training.md

```diff
@@ -0,0 +1,184 @@
+
+# Quantization aware training
+
+In quantization aware training, we train the model with quantization in the loop
+and ensure that the forward pass matches precision for both training and inference. There are two key aspects to this:
+<ol>
+<li> Operator fusion at inference time needs to be accurately modeled at training time </li>
+<li> Quantization effects at inference are modeled at training time </li>
+</ol>
+
+
+## Quantization aware training with TensorFlow
+For efficient inference, we combine batch normalization with preceding convolutional/fully connected layers prior to quantization by [folding batch norm layers](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/python/fold_batch_norms.py). 
+
+We model quantization error by inserting [fake quantization](../api_guides/python/array_ops.md#Fake_quantization) nodes simulate the
+effect of quantization in the forward and backward passes. The forward pass models quantization, while the backward pass models quantization as a straight through estimator (See [Bengio et al.,2013](https://arxiv.org/abs/1308.3432)). Both the forward and backward pass simulate the quantization of weights and activations. (See [Benoit et al.,2017](https://arxiv.org/abs/1712.05877), [Krishnamoorthi,2018](https://arxiv.org/abs/1806.08342))
```

Comment:
In general, we're writing engineering docs and not academic docs. Do these papers help users with this? Can you summarize why they would care?